lesson5_task4.py

```diff
@@ -0,0 +1,27 @@
+"""
+The math quiz program
+
+Write a program that asks the answer for a mathematical expression, checks whether the user is right or wrong,
+and then responds with a message accordingly.
+"""
+import random
+
+# Generate random numbers to work with
+num1 = random.randint(1, 100)
+num2 = random.randint(1, 100)
+
+# Generate random simple operator
```

Comment:
Very nice, you could try to add some more operators, but in general, everything looks nice, you code style is very readable and easy to comprehend, keep it this way :)